MinecraftClient/McClient.cs

```diff
@@ -2066,8 +2068,17 @@ public void OnLatencyUpdate(Guid uuid, int latency)
             string playerName = null;
             if (onlinePlayers.ContainsKey(uuid))
             {
+                Dictionary<int, Entity> keyValuePairs = GetEntities();
                 playerName = onlinePlayers[uuid];
                 playersLatency[playerName] = latency;
+                foreach (KeyValuePair<int, Entity> ent in keyValuePairs)

```

Comment:
(2/2)... so here the loop becomes `foreach (KeyValuePair<int, Entity> ent in entities)`